runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/config/KafkaTopicConfigAdapter.java

```diff
@@ -41,9 +37,10 @@ public final class KafkaTopicConfigAdapter implements JsonbAdapter<KafkaTopicCon
     private static final String EVENT_KEY = "key";
     private static final String EVENT_VALUE = "value";
     private static final String SUBJECT = "subject";
-    private static final String HEADERS_NAME = "headers";
+    private static final String TRANSFORMS_NAME = "transforms";
 
     private final ModelConfigAdapter converter = new ModelConfigAdapter();
+    private final KafkaTopicTransformsConfigAdapter transformsConverter = new KafkaTopicTransformsConfigAdapter();
```

Comment:
```suggestion
    private final ModelConfigAdapter model = new ModelConfigAdapter();
    private final KafkaTopicTransformsConfigAdapter transforms = new KafkaTopicTransformsConfigAdapter();
```

These are not converters (means something else for us too) plus probably no need to include the type in the name.